### DISCUSSION TITLE
Project name change

### DISCUSSION THREAD

**Discussion by earth4s** [kagemomiji/airsonic-advanced#448] [General]
Seeing as this is another successor to SubSonic/Airsonic/Airsonic-Advanced, Has any thought been given to changing the name to differentiate it from these previous versions? It might help when people are searching/mentioning/discussing  this particular branch.

**Comment by bartmans** [comment]
I think there are discussions about merging the repo from Airsonic Advanced with this one. So this will then be the 'official' Airsonic Advanced repository.
